src/app/charts/chart.component.ts

```diff
@@ -51,7 +51,10 @@ export class ChartComponent implements OnChanges {
             indicator: this.chart.indicator,
             scenario: this.scenario,
             city: this.city,
-            climateModels: this.models
+            climateModels: this.models,
+            // As a temporary solution, the time agg defaults to the 1st valid option.
```

Comment:
Do we have a github issue to add selecting non-yearly granularities? That'd be pretty handy to have